packages/syft/src/syft/client/client.py

```diff
@@ -366,6 +371,7 @@ def api(self) -> SyftAPI:
 
     def guest(self) -> Self:
         self.credentials = SyftSigningKey.generate()
+        self.__logged_in_user = ""
```

Comment:
cc: @shubham3121   , yash found a nice way to prevent inputting email address without a service endpoint by adding email as an attribute during login.
Pinging for comment , as we had discussed on this.